tfx/components/example_gen/component.py

```diff
@@ -93,10 +93,8 @@ def __init__(self,
     output_config = output_config or utils.make_default_output_config(
         input_config)
     if not example_artifacts:
-      artifact = standard_artifacts.Examples()
-      artifact.split_names = artifact_utils.encode_split_names(
-          utils.generate_output_split_names(input_config, output_config))
-      example_artifacts = channel_utils.as_channel([artifact])
+      example_artifacts = channel_utils.as_channel(
+          [standard_artifacts.Examples()])
```

Comment:
@1025KB I vaguely remember there is a reason that we cannot drop the artifact yet. Is it for cloud?

/cc @numerology